Makefile

```diff
@@ -287,7 +287,7 @@ react-build:
 .PHONY: frontend-fast
 # Build frontend into static files faster by setting BUILD_AS_FAST_AS_POSSIBLE=true flag, which disables eslint and typechecking.
 frontend-fast:
-	cd frontend/ ; yarn run buildFast
+	cd frontend/ ; yarn run build
```

Comment:
we also have 

```Makefile
.PHONY: frontend
# Build frontend into static files.
frontend: react-build
```
and I am wondering whether we can get rid of that with Vite and merge it with `frontend-fast`

Reply:
Yes. We can probably remove the frontend-fast because `make frontend` is fast enough. Will look into removing that.

Reply:
Found a way to improve frontend build performance, so keeping the `make frontend-fast`